tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp

```diff
@@ -469,8 +469,8 @@ namespace tcf {
                     "Signature verification of client request failed. Request is tampered.");
             }
             std::vector<tcf::WorkOrderData> wo_data = ExecuteWorkOrder();
-            int i = 0;
-            int out_data_size = data_items_out.size();
+            size_t i = 0;
+            size_t out_data_size = data_items_out.size();
```

Comment:
Change in other places in the same file.

Reply:
Will do.